src/django/pfb_analysis/aws_batch.py

```diff
@@ -0,0 +1,36 @@
+import logging
+
+import boto3
+from botocore.exceptions import BotoCoreError
+
+
+logger = logging.getLogger(__name__)
+
+
+class NoActiveJobDefinitionRevision(BotoCoreError):
+    """ Raised when an AWS Batch Job Definition has no active revision
+
+    Init with kwarg 'job_definition' to display in error
+    e.g. raise NoActiveJobDefinitionError(job_definition='foo')
+
+    """
+    fmt = '{job_definition}'
+
+
+def get_latest_job_definition(job_definition_name):
+    """ Get the latest revision of an AWS Batch job definition
+
+    Raises NoActiveJobDefinitionRevision if no current active revision for the
+        requested job definition
+
+    """
+    client = boto3.client('batch')
+    response = client.describe_job_definitions(jobDefinitionName=job_definition_name,
+                                               status='ACTIVE')
+    # TODO: Continue building the array if more than 100 results
```

Comment:
A little worried about this falling through the cracks until it catches us by surprise.  What about either
```
    while(response.get('nextToken') is not None):
        response = client.describe_job_definitions(jobDefinitionName=job_definition_name,
                                                   status='ACTIVE', nextToken=response['nextToken'])
        job_definitions.extend(response.get('jobDefinitions', []))
```
or raising an exception if `nextToken` has a value, so we'll be alerted to the fact that we've hit the 100-definition mark?